packages/core/status/core-status-server-internal/src/status_service.test.ts

```diff
@@ -82,7 +82,7 @@ describe('StatusService', () => {
       expect(prebootRouterMock.get).toHaveBeenCalledWith(
         {
           path: '/api/status',
-          options: { authRequired: false, tags: ['api'] },
+          options: { authRequired: false, tags: ['api'], access: 'public' },
```

Comment:
A comment explaining our reasoning is a way we can share it with the dev community.